pkg/mount-manager/safe-mounter-v1_windows.go

```diff
@@ -337,3 +337,17 @@ func (mounter *CSIProxyMounterV1) MountSensitiveWithoutSystemd(source string, ta
 func (mounter *CSIProxyMounterV1) MountSensitiveWithoutSystemdWithMountFlags(source string, target string, fstype string, options []string, sensitiveOptions []string, mountFlags []string) error {
 	return errors.New("MountSensitiveWithoutSystemd is not implemented")
 }
+
+// CanSafelySkipMountPointCheck always returns false on Windows.
```

Comment:
@msau42 I had to add this because of two breaking changes that updated the interface. Could you confirm that it's safe to use the upstream implementation?

https://github.com/kubernetes/mount-utils/blob/master/mount_windows.go#L247-L259

Reply:
Until we switch to Windows host process https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1071, we may not be able to do many mount operations directly. @mauriciopoppe can you confirm?

Reply:
This file only uses the following exported properties from k8s.io/mount-utils:

- mount.NormalizeWindowsPath (method)
- mount.MountPoint (struct)

None of these paths use these two methods. I believe it's safe to use the upstream implementation, in Windows we're not doing mount directly yet because #1071 isn't merged, if #1071 is merged these two paths won't be used still.

lgtm